Rakefile

```diff
@@ -19,7 +19,7 @@ RSpec::Core::RakeTask.new(:spec)
 
 desc "Run RSpec tests that don't require VM fixtures or a particular shell"
 RSpec::Core::RakeTask.new(:unit) do |t|
-  t.rspec_opts = '--tag ~ssh --tag ~docker --tag ~bash --tag ~winrm ' \
+  t.rspec_opts = '--tag ~ssh --tag ~docker --tag ~docker_wcow --tag ~bash --tag ~winrm ' \
```

Comment:
@nicklewis does this new `docker_wcow` tag jive with you recently cleaned things up?

Reply:
Yeah, this fits. The `travisci` job is for inexpensive tests / tests that don't depend on expensive-to-build containers and the `puppetserver` job is for the expensive tests / tests with expensive dependencies. Those jobs correspond to the "unit" and "integration" cells in the travis matrix.